src/main/java/core/basesyntax/model/Transaction.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.model;
+
+public class Transaction {
+    private Operation operation;
+    private String fruitName;
+    private int quantity;
+
+    public Transaction(String operation, String fruitName, int quantity) {
```

Comment:
remove logic from the constructor
```suggestion
    public Transaction(Operation operation, String fruitName, int quantity) {
```